cocotb/BUILD

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 Google LLC
```

Comment:
Should change the copyright header to the true owner. Either yourself or the company you represent.